robomo-discord-bot/ecs-code/index.ts

```diff
@@ -92,17 +100,19 @@ async function main() {
     }
   });
 
-  const discordToken = await getSecret('DiscordBotToken');
-  await client.login(discordToken);
+  const discordToken: string = await getSecret('DiscordBotToken');
+  const tokenJson: DiscordToken = JSON.parse(discordToken) as DiscordToken;
```

Comment:
Are the values of your secrets JSON? the secrets that I added to the prod account are just strings.

Reply:
oh I think I had entered the secrets as key-value pairs in my account, which is probably why it was getting returned as JSON. I can revert this change here, just a confusion on my part